src/components/mission.css

```diff
@@ -0,0 +1,208 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500&display=swap');
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.loader {
+  margin: 20px auto;
+  color: #212a3e;
+  position: relative;
+  display: inline-block;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 48px;
+  letter-spacing: 4px;
+  box-sizing: border-box;
+}
+
+.loader::before {
+  content: '';
+  position: absolute;
+  right: 70px;
+  bottom: 10px;
+  height: 28px;
+  width: 5.15px;
+  background: currentColor;
+  box-sizing: border-box;
+  animation: animloader1 1s linear infinite alternate;
+}
+
+.loader::after {
+  content: '';
+  width: 10px;
+  height: 10px;
+  position: absolute;
+  left: 125px;
+  top: 2px;
+  border-radius: 50%;
+  background: red;
+  box-sizing: border-box;
+  animation: animloader 1s linear infinite alternate;
+}
+
+@keyframes animloader {
+  0% {
+    transform: translate(0, 0) scaleX(1);
+  }
+
+  14% {
+    transform: translate(-12px, -16px) scaleX(1.05);
+  }
+
+  28% {
+    transform: translate(-27px, -28px) scaleX(1.07);
+  }
+
+  42% {
+    transform: translate(-46px, -35px) scaleX(1.1);
+  }
+
+  57% {
+    transform: translate(-70px, -37px) scaleX(1.1);
+  }
+
+  71% {
+    transform: translate(-94px, -32px) scaleX(1.07);
+  }
+
+  85% {
+    transform: translate(-111px, -22px) scaleX(1.05);
+  }
+
+  100% {
+    transform: translate(-125px, -9px) scaleX(1);
+  }
+}
+
+@keyframes animloader1 {
+  0% {
+    box-shadow: 0 -6px, -122.9px -8px;
+  }
+
+  25%,
+  75% {
+    box-shadow: 0 0, -122.9px -8px;
+  }
+
+  100% {
+    box-shadow: 0 0, -122.9px -16px;
+  }
+}
+
+.table {
+  overflow-y: hidden;
+  border-collapse: collapse;
+  border: 1px solid black;
+  max-width: 1280px;
+  width: 94%;
+  margin: 30px auto;
+
+  /* display: flex; */
```

Comment:
Kindly take away every commented piece of code in your codebase as it makes it messy and gives it a less professional look😉.